common/reviews/api/rush-lib.api.md

```diff
@@ -670,7 +670,10 @@ export interface IPhasedCommand extends IRushCommand {
 
 // @public
 export interface IPnpmLockfilePolicies {
-    disallowInsecureSha1?: boolean;
+    disallowInsecureSha1?: {
+        enabled: boolean;
+        exmeptPackageVersions: Record<string, Array<string>>;
```

Comment:
The spelling is incorrect.  It should be `exemptPackageVersions`.

Also, instead of `Array<string>` use `string[]` because it is idiomatic.